types/picomatch/lib/picomatch.d.ts

```diff
@@ -0,0 +1,77 @@
+import constantsImport = require('./constants');
+import parseImport = require('./parse');
+import scanImport = require('./scan');
+import { PicomatchOptions as PicomatchOptionsImport } from './picomatch-options';
+
+/**
+ * Creates a matcher function from one or more glob patterns. The
+ * returned function takes a string to match as its first argument,
+ * and returns true if the string is a match. The returned matcher
+ * function also takes a boolean as the second argument that, when true,
+ * returns an object with additional information.
+ *
+ * ```js
+ * const picomatch = require('picomatch');
+ * // picomatch(glob[, options]);
+ *
+ * const isMatch = picomatch('*.!(*a)');
+ * console.log(isMatch('a.a')); //=> false
+ * console.log(isMatch('a.b')); //=> true
+ * ```
+ * @param glob One or more glob patterns.
+ * @return Returns a matcher function.
+ * @api public
+ */
+declare function picomatch(
+    glob: string | string[],
+    options?: picomatch.PicomatchOptions,
+    returnState?: boolean,
+): picomatch.Matcher;
+
+declare namespace picomatch {
+    interface Matcher {
+        (test: string): boolean;
+    }
+
+    type PicomatchOptions = PicomatchOptionsImport;
```

Comment:
Not required IMHO

Reply:
In the part of the review that you may not have been looking at, I'm using this type in the `rrdir` .d.ts files. This was really the reason I touched anything at all in `picomatch` in the first place. On this second round (after review) it seemed to me like it would be better to expose this type through the single point of export rather than manually grabbing the file containing this options type. I suspect that other libraries may also want to have access to this type as I did in `rrdir` since the implementation there was tied to `picomatch`.